src/main/steps/applicant1/connection-summary/content.ts

```diff
@@ -106,6 +145,7 @@ const en = ({ isDivorce, partner, userCase }: CommonContent) => {
     readMore: 'Read more about your connections',
     ...enContainsHabitualResConnection(userCase?.connections),
     ...enContainsDomConnection(userCase?.connections),
+    ...getResidual(isDivorce),
```

Comment:
shouldn't this be '...enContainsResidualConnection(userCase?.connections, isDivorce)' ?